ggezchain/chain.json

```diff
@@ -0,0 +1,188 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "ggezchain",
+  "chain_type": "cosmos",
+  "chain_id": "ggezchain",
+  "website": "https://ggez.one/",
+  "pretty_name": "GGEZ1 Chain",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "ggez",
+  "daemon_name": "ggezchaind",
+  "node_home": "$HOME/.ggezchain",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uggez1",
+        "fixed_min_gas_price":  0.2,
+        "low_gas_price":  0.4,
+        "average_gas_price": 0.5,
+        "high_gas_price": 0.75
+      },
+      {
+        "denom": "uggz",
+        "fixed_min_gas_price":  0.25,
+        "low_gas_price":  0.5,
+        "average_gas_price": 0.75,
+        "high_gas_price": 1
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uggez1"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/GGEZLabs/ggezchain",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": [
+      "v1.0.0"
+    ],
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.12"
+    },
+    "binaries": {
+      "linux/amd64": "https://github.com/GGEZLabs/ggezchain/releases/download/v1.0.0/ggezchaind"
+    },
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/GGEZLabs/ggez-mainnet/refs/heads/main/genesis/genesis.json" 
+    },
+    "versions": [
```

Comment:
Please relocate `"versions"` to a separate versions.json file. Version history is no longer described in the chain.json